code/datums/tutorial/marine/role_specific/hospital_corpsman_basic.dm

```diff
@@ -0,0 +1,1090 @@
+	manual_message_to_player("A small overdose, while annoying, will very rarely prove lethal in the body.")
+
+	register_tutorial_step(CALLBACK(src, PROC_REF(od_tutorial_4)))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_basic/proc/od_tutorial_4()
+	SIGNAL_HANDLER
+
+	manual_message_to_player("The effects of every overdose will have different methods of treatment depending on the chemical.")
+	manual_message_to_player("To stabilize a patient after an overdose, you should follow the automatic medicine recommendations displayed on the bottom of your <b>Health Analyzer</b> interface (you may need to scroll down to see them).")
+	manual_message_to_player("As you can see on your Health Analyzer scan, the Bicaridine overdose is creating Burn Damage on the Dummy.")
+	manual_message_to_player("Kelotane, which is used to primarily treat <b>Burn</b> damage, can also be used to counteract the effects of this overdose.")
+	manual_message_to_player("We will follow the Health Analyzers recommendations on the bottom of the interface, and feed the Dummy a <b>Kelotane Pill</b>.")
+
+	register_tutorial_step(CALLBACK(src, PROC_REF(od_tutorial_5)))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_basic/proc/od_tutorial_5()
+	SIGNAL_HANDLER
+
+	handle_pill_bottle(marine_dummy, "Kelotane", "Kl", "2", /obj/item/reagent_container/pill/kelotane)
+
+	RegisterSignal(tutorial_mob, COMSIG_MOB_TUTORIAL_HELPER_FAIL, PROC_REF(od_tutorial_5), TRUE)
+	RegisterSignal(tutorial_mob, COMSIG_MOB_TUTORIAL_HELPER_RETURN, PROC_REF(od_tutorial_6))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_basic/proc/od_tutorial_6()
+	SIGNAL_HANDLER
+
+	UnregisterSignal(tutorial_mob, COMSIG_MOB_TUTORIAL_HELPER_RETURN)
+	UnregisterSignal(tutorial_mob, COMSIG_MOB_TUTORIAL_HELPER_FAIL)
+
+	manual_message_to_player("Well done, the Dummy's condition is now stable, and their overdose will disappear over time.")
+
+	register_tutorial_step(CALLBACK(src, PROC_REF(tutorial_close)))
+
+// Helpers
+
+/**
+* Handles the creation and describes the use of pill bottles and pills in HM tutorials
+*
+* NOT FOR USE OUTSIDE OF TUTORIAL SYSTEM
+*
+* Currently limited to /mob/living/carbon/human/realistic_dummy
+*
+* Will break if used more than once per proc, see add_to_tracking_atoms() limitations
+*
+* Arguments:
+* * target Set to realistic_dummy of choosing
+* * name Uppercased name of the selected chemical, in string form
+* * maptext Sets the maptext_label variable for the created pill bottle, also a string
+* * iconnumber Sets the icon for the created pill bottle, input a number string ONLY (IE: "1")
+* * pill Typepath of the pill to place into the pill bottle
+*/
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_basic/proc/handle_pill_bottle(target, name, maptext, iconnumber, pill)
```

Comment:
Eating the pill instead of giving it to the dummy will cause there to be two bottles for a short time